src/mmw/js/src/core/models.js

```diff
@@ -217,8 +217,10 @@ var TaskMessageViewModel = Backbone.Model.extend({
     message: null,
     iconClass: null,
 
-    setError: function() {
-        this.set('message', 'Error');
+    setError: function(err) {
+        var message = err === 'NO_LAND_COVER' ? 'Selected area of interest ' +
+            'doesn\'t include enough land cover to run the model' : 'Error';
```

Comment:
Instead of saying "enough" (which implies some unstated magic threshold that the user must guess), we should say "any" because that is what we are checking for. The caught areas of interest don't have _any_ land cover.